tools/backups2datalad.py

```diff
@@ -446,13 +448,35 @@ def get_file_bucket_url(self, asset: RemoteAsset) -> str:
 
     def tag_releases(self, dandiset: RemoteDandiset, ds: Dataset, push: bool) -> None:
         log.info("Tagging releases for Dandiset %s", dandiset.identifier)
-        for v in dandiset.get_versions():
-            if v.identifier != "draft":
-                if readcmd("git", "tag", "-l", v.identifier, cwd=ds.path):
-                    log.debug("Version %s already tagged", v.identifier)
-                else:
-                    log.info("Tagging version %s", v.identifier)
-                    self.mkrelease(dandiset.for_version(v), ds, push=push)
+        versions = [v for v in dandiset.get_versions() if v.identifier != "draft"]
+        for v in versions:
+            if readcmd("git", "tag", "-l", v.identifier, cwd=ds.path):
+                log.debug("Version %s already tagged", v.identifier)
+            else:
+                log.info("Tagging version %s", v.identifier)
+                self.mkrelease(dandiset.for_version(v), ds, push=push)
+        if versions:
+            latest = max(map(attrgetter("identifier"), versions), key=Version)
+            description = readcmd(
+                "git", "describe", "--tags", "--long", "--always", cwd=ds.path
+            )
+            if "-" not in description:
+                # No tags on master branch
+                merge = True
+            else:
+                m = re.fullmatch(
+                    r"(?P<tag>.+)-(?P<distance>[0-9]+)-g(?P<rev>[0-9a-f]+)?",
+                    description,
+                )
+                assert m, f"Could not parse `git describe` output: {description!r}"
+                merge = Version(latest) > Version(m["tag"])
+            if merge:
+                log.debug("Running: git merge -s ours %s", shlex.quote(latest))
+                subprocess.run(
+                    ["git", "merge", "-s", "ours", latest], cwd=ds.path, check=True
```

Comment:
I have tried it merged into #68 on 000029 and got an editor suggesting to edit the merge commit message. Please disable interactions by providing a merge message to `merge` invocation .  Also make that merge message mention that it is "no change" merge, e.g.

```suggestion
                    ["git", "merge", "-s", "ours", "-m", f"Merge '{latest}' into the branch with drafts (no differences in content merged)", latest], cwd=ds.path, check=True
```

or alike.